app/View/Names/fields.inc

```diff
@@ -171,7 +171,7 @@
             print $this->Form->error('Name.type');
           }
         } else {
-          print $vv_available_types[ $names[0]['Name']['type'] ];
+          print _txt('en.name.type', null, $names[0]['Name']['type']);
```

Comment:
What's the reason for this change? `$vv_available_types` is the "newer" way to pass localization strings.

Reply:
Reverted the change